packages/kbn-search-connectors/components/sync_jobs/documents_panel.tsx

```diff
@@ -24,21 +25,66 @@ export const SyncJobDocumentsPanel: React.FC<SyncJobDocumentsPanelProps> = (sync
   const columns: Array<EuiBasicTableColumn<SyncJobDocumentsPanelProps>> = [
     {
       field: 'added',
-      name: i18n.translate('searchConnectors.index.syncJobs.documents.added', {
-        defaultMessage: 'Upserted',
-      }),
+      name: (
+        <EuiToolTip
+          content={
+            <FormattedMessage
+              id="searchConnectors.index.syncJobs.documents.upserted.tooltip"
+              defaultMessage="The number of {index} operations the connector sent to the Elasticsearch _bulk API during this sync. This includes net-new documents and updates to existing documents. This does not account for duplicate _ids, or any documents dropped by an ingest processor"
```

Comment:
"this does not account for duplicate _ids" is a bit ambiguous to me.

Reply:
What I'm meaning to indicate is that if I send 3 docs like:
```
{"_id": "foo", "text": "bar"}
{"_id": "foo", "text": "baz"}
{"_id": "foo", "text": "qux"}
```
We'll count this as `3 Upserted`, but your index will have 1 document.

Suggestions on how to make that clear and concise?

Reply:
"Duplicate _ids, error responses and documents dropped by an ingest processor can cause discrepancies with the number of documents in the index."

But then it's longer again